apps/omg_eth/lib/omg_eth/ethereum_height.ex

```diff
@@ -46,21 +45,20 @@ defmodule OMG.Eth.EthereumHeight do
     {:reply, {:ok, ethereum_height}, ethereum_height}
   end
 
-  def handle_info({:internal_event_bus, :newHeads, new_heads}, state) do
-    value = new_heads["params"]["result"]["number"]
-
-    case is_binary(value) do
+  def handle_info({:internal_event_bus, :ethereum_new_height, new_height}, state) do
+    case is_integer(new_height) do
       true ->
-        ethereum_height = Encoding.int_from_hex(value)
-        _ = Logger.debug("Got a newHeads event for new Ethereum height #{ethereum_height}.")
+        _ = Logger.debug("Got an internal :ethereum_new_height event with height: #{new_height}.")
 
         _ =
-          if rem(ethereum_height, 1000) == 0,
-            do: Logger.info("Got a newHeads event for new Ethereum height #{ethereum_height}. (log every 1000)")
+          if rem(new_height, 1000) == 0 do
```

Comment:
possibly we'll want to remove this (i.e. the `log every 1000`). It was specifically added in (2x) to debug the something suspected to be a ws subscription problem.

If not immediately, then a `TODO` or issue maybe?